torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -101,6 +101,73 @@ class _X86InductorQuantizationAnnotation(QuantizationAnnotation):
 QUANT_ANNOTATION_KEY = "quantization_annotation"
 
 
+def _skip_annotate(
+    nodes: List[Node], filter_fn: Optional[Callable[[Node], bool]] = None
+):
+    # 1) Skip annotate if any node is already annotated
+    if _is_any_annotated(nodes):
+        return True
+
+    # 2) Not skip annotate if a) filter_fn is provided and b) any node passed the filter
+    if filter_fn and any(filter_fn(node) for node in nodes):
+        return False
+
+    return True
+
+
+def _get_operator_type_filter(operator_type: Callable, module_name_list):
+    """Get the operator_type_filter function for a given operator type and module name list.
+
+    The filter accept a node and checks if 1) the node has certain operator type,
+    and 2) the node does not marked by `set_module_name_qconfig`.
+
+    For example:
+        # linear_op.target if torch.ops.aten.linear.default
+        # linear_op is traced from `fc3`
+        node: linear_op = call_function[...](...)
+
+
+    >> operator_type_filter = _get_operator_type_filter(torch.ops.aten.linear.default, ["fc1", "fc2"])
+    >> print(operator_type_filter(node))
+    True  # the node's target is `torch.ops.aten.linear.default` and not marked by `set_module_name_qconfig`
+    """
+    module_name_list_filters = [_get_module_name_filter(m) for m in module_name_list]
+
+    def operator_type_filter(n: Node) -> bool:
+        not_module_name_node = not any(f(n) for f in module_name_list_filters)
+        has_certain_operator_type = n.target == operator_type
+        return not_module_name_node and has_certain_operator_type
+
+    return operator_type_filter
+
+
+def _get_not_operator_type_or_name_filter(
+    tp_list: List[torch._ops.OpOverloadPacket], module_name_list: List[str]
```

Comment:
Similar as above, we don't need `tp_list` and `module_name_list` as inputs. We may only need to ensure it's in the default quantizable op list.

Reply:
Yes, updated.